app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::ReservationsController < ApplicationController
+  # GET /reservations
+  def index
+    @reservations = Reservation.all
+
+    render json: @reservations, include: :bike
+  end
+
+  # GET /reservations/1
+  def show
+    render json: @reservation, include: :bike
+  end
```

Comment:
- Please the func to find the reservation before rendering. 🐱

Reply:
yup you are right :+1: .. now its working and already modify the controllers..

![Screenshot from 2023-01-20 05-50-36](https://user-images.githubusercontent.com/104095543/213594562-5a5ec453-e88e-45ca-b284-8d60948bbebb.png)